src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -23,17 +23,6 @@
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.TimeoutException;
 
-import org.folio.circulation.domain.ItemStatus;
```

Comment:
Please can these imports be ordered according to https://issues.folio.org/browse/FOLIO-1667

Reply:
I have the same issue with imports as in my previous PR. My bad. I configured my IDE manually
Just in case. Are these requirements for imports order?
```
No use of wildcard import statements
Static imports in alphabetic order first
Then imports in defined then alphabetic order (java, javax, org, com, everything else) with a line break between defined groups
```

Reply:
Yes, that is the policy (it matches the eclipse behaviour). I uploaded an IntelliJ config that can be imported [here](https://issues.folio.org/secure/attachment/16722/Eclipse%20style%20imports.xml)

Reply:
Done